ember-cli-build.js

```diff
@@ -55,5 +46,31 @@ module.exports = function (defaults) {
     },
   });
 
-  return app.toTree();
+  return compat.compatBuild(app, Webpack, {
+    packagerOptions: {
+      webpackConfig: {
+        ...rdfaEditorWebpackConfig,
```

Comment:
I think it would be better to use `webpack-merge` in this case, in order to ensure we do not override the editor-config. Additionally it would be better to import the editor-plugins config (which already includes the editor-config)

Reply:
I didn't know either of those existed, I've updated now.